string address
int16 rfcomm_port
string node_name
